mysql-test/suite/rocksdb/t/type_char_secondary_index_only_scan.test

```diff
@@ -0,0 +1,50 @@
+# 1. Find all collations that support index-only scans with secondary
+#    keys using the "Extra" field from "EXPLAIN SELECT"
+# 2. Verify if the original string is reconstructed correctly from SK
+#    (or from PK when index-only scans with SK are not supported)
```

Comment:
I didn't see accessing the PK when SK is not supported. In order to guarantee the correctness of unpack_info, we should ideally rely of PK rather than SK because SK can have weirdness of covering and non covering.

Reply:
When `EXPLAIN SELECT` returns `Using index` it means that SK was used.
When `EXPLAIN SELECT` returns `NULL` we know that PK was used because MyRocks switches to PK if SK can't be used.